mount/titus-mount-nfs.c

```diff
@@ -331,7 +340,7 @@ int main(int argc, char *argv[])
 	/* Now we can do the fs_config calls and actual mount */
 	do_fsconfigs(fsfd, final_options);
 	mkdir_p(target);
-	mount_and_move(fsfd, target, pidfd, flags_ul);
+	mount_and_move(fsfd, target, flags_ul);
```

Comment:
Haha, nice, we were not even using pidfd anyway

Reply:
yup